lib/oidcstrategy.js

```diff
@@ -45,7 +45,6 @@ const InternalOpenIDError = require('./errors/internalopeniderror');
 const Log = require('./logging').getLogger;
```

Comment:
Please clean up the capitalization for these consts. They should all be lower case or all be upper case.